canonical-web-and-design/maas-ui
# refactor(ui): move logs components to base

## Done

- Move the machine logs components to the base folder.
- Migrate to React Testing Library.
Note: the components haven't been updated to worth controllers yet, that will come in https://github.com/canonical-web-and-design/app-tribe/issues/944.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the Logs tab for a machine and check you can see logs, events and download the files.

## Fixes

Fixes: canonical-web-and-design/app-tribe#1008.
